src/native/libs/System.IO.Compression.Native/CMakeLists.txt

```
@@ -18,6 +18,9 @@ if (NOT CLR_CMAKE_TARGET_BROWSER AND NOT CLR_CMAKE_TARGET_WASI)
 
     if (CLR_CMAKE_USE_SYSTEM_BROTLI)
         add_definitions(-DFEATURE_USE_SYSTEM_BROTLI)
+        if (CLR_CMAKE_TARGET_FREEBSD)
```

Check host here instead of target?

```suggestion
        if (CLR_CMAKE_HOST_FREEBSD)
```